src/utils/rest.ts

```diff
@@ -45,10 +49,13 @@ export function post<TBody extends Record<string, string | number | boolean> = {
   url: string,
   secretApiKey: string,
   body: TBody,
+  addJsonHeader = false,
```

Comment:
Can this be updated to pass in custom headers?

Reply:
Yep good call, updated in latest